src/store/embalm/actions.ts

```diff
@@ -19,8 +19,15 @@ export enum ActionType {
   SetArchaeologistPublicKey = 'EMBALM_SET_ARCHAEOLOGIST_PUBLIC_KEY',
   SetArchaeologistSignature = 'EMBALM_SET_ARCHAEOLOGIST_SIGNATURE',
   SetArchaeologists = 'EMBALM_SET_ARCHAEOLOGISTS',
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  setShowSelectedArchaeologists = 'EMBALM_SET_SHOW_SELECTED_ARCHAEOLOGISTS',
```

Comment:
Rename this to `SetShowSelectedArchaeologists` and you should be able to remove the eslint comment.